jscomp/main/jsoo_main.ml

```diff
@@ -167,15 +167,15 @@ let make_compiler name impl =
                     Js.wrap_meth_callback
                       (fun _ code -> (shake_compile impl ~use_super_errors:true (Js.to_string code)));
                     "version", Js.Unsafe.inject (Js.string (Bs_version.version));
-                    "load_module",
+                    (* "load_module",
```

Comment:
@bobzhang After these changes, it seems that the list of modules need to be known at compile time, is there a way to defer loading cmi/cmj files at runtime from the resulting JS playground, like `load_module` did before?